src/main/java/org/opentripplanner/apis/transmodel/mapping/RequestModesMapper.java

```diff
@@ -8,22 +8,25 @@
 class RequestModesMapper {
 
   /**
-   * Maps a GraphQL Modes input type to a RequestModes.
-   *
-   * This only maps access, egress, direct & transfer.
-   * Transport modes are now part of filters.
-   * Only in case filters are not present we will use this mapping
+   * Maps GraphQL Modes input type to RequestModes.
+   * <p>
+   * This only maps access, egress, direct & transfer modes. Transport modes are set using filters.
+   * Default modes are WALK for access, egress, direct & transfer.
    */
-  @SuppressWarnings("unchecked")
   static RequestModes mapRequestModes(Map<String, ?> modesInput) {
-    StreetMode accessMode = (StreetMode) modesInput.get("accessMode");
-    RequestModesBuilder mBuilder = RequestModes
-      .of()
-      .withAccessMode(accessMode)
-      .withEgressMode((StreetMode) modesInput.get("egressMode"))
-      .withDirectMode((StreetMode) modesInput.get("directMode"));
+    RequestModesBuilder mBuilder = RequestModes.of();
 
-    mBuilder.withTransferMode(accessMode == StreetMode.BIKE ? StreetMode.BIKE : StreetMode.WALK);
+    if (modesInput.containsKey("accessMode")) {
```

Comment:
Can you move these strings into constants, now that you're using them twice?

Reply:
Done.